pkg/apis/servicecatalog/validation/broker.go

```diff
@@ -90,7 +90,7 @@ func validateClusterServiceBrokerSpec(spec *sc.ClusterServiceBrokerSpec, fldPath
 	commonErrs := validateCommonServiceBrokerSpec(&spec.CommonServiceBrokerSpec, fldPath)
 
 	if len(commonErrs) != 0 {
-		allErrs = append(commonErrs)
+		allErrs = append(allErrs, commonErrs...)
```

Comment:
Doh! I'm saddened that go didn't warn you about that! Silly go...

Reply:
I was surprised as well, wondering when that behavour would even be desired? Tricky...

Reply:
I am guessing it's the equivalent of this

```golang
allErrs = append(allErrs, []error{}...)
```